mentor_classifier/mentor_classifier/arch/lr-transformer/embeddings.py

```diff
@@ -0,0 +1,20 @@
+#
+# This software is Copyright ©️ 2020 The University of Southern California. All Rights Reserved.
+# Permission to use, copy, modify, and distribute this software and its documentation for educational, research and non-profit purposes, without fee, and without a written agreement is hereby granted, provided that the above copyright notice and subject to the full license file found in the root of this software deliverable. Permission to make commercial use of this software may be obtained by contacting:  USC Stevens Center for Innovation University of Southern California 1150 S. Olive Street, Suite 2300, Los Angeles, CA 90115, USA Email: accounting@stevens.usc.edu
+#
+# The full terms of this copyright and license should always be found in the root directory of this software deliverable as "license.txt" and if these terms are not found with this software, please contact the USC Stevens Center for the full license.
+#
+from mentor_classifier.sentence_transformer import find_or_load_sentence_transformer
+from sentence_transformer import SentenceTransformer
+
+
+class TransformerEmbeddings:
+    def __init__(self, shared_root):
+        self.transformer: SentenceTransformer = find_or_load_sentence_transformer(
```

Comment:
use path.join and we should try to find a way to not need the backstep?

```python
find_or_load_sentence_transformer(os.path.join(shared_root, "sentence-transformer")
```

...also will save us a lot of headache if we can add a file ext to the pretrained model, like `sentence-transformer.bin`...if we don't have that, will be harder to distinguish that file as going to git-lfs elsewhere in the site deployment for the app
```

looking below, maybe the files are really already .pkl? if that's the case, ignore this